src/AaveV2EthBUSDIR_20230602/BUSD-OFFBOARDING-PLAN-PART-II-AIP.md

```diff
@@ -0,0 +1,81 @@
+---
+title: BUSD Offboarding Plan Part II
+author: Marc Zeller (@marczeller), Aave-Chan initiative
+shortDescription: BUSD Offboarding Plan Part II
+discussions: https://governance.aave.com/t/arfc-busd-offboarding-plan-part-ii/13048
+created: 2022-06-02
+---
+
+## Simple Summary
+
+This AIP proposes to change the InterestRate Strategy the BUSD reserve on the Aave V2 Ethereum pool and withdraw POL aBUSD from Aave V2 Ethereum Pool
+
+## Motivation
+
+[Please refers to the first part of the BUSD offboarding plan for more context.](https://governance.aave.com/t/arfc-busd-offboarding-plan/12170)
+
+The first part of the plan was [a success](https://governance.aave.com/t/arfc-busd-offboarding-plan/12170/3?u=marczeller), driving out liquidity and incentivizing active users to repay their debt and move their position to other stablecoins.
+
+The remaining vBUSD debt holders seem to be inactive or unaffected by high rates, due to the time-sensitive nature of BUSD/Paxos situation and the potential risk of protocol bad debt.
+
+This second part of the offboarding plan will concentrate efforts on creating unsustainable positions for remaining borrowers to either motivate them enough to repay or reach liquidation thresholds.
+
+Both actions are proposed to be performed :
+
+1. modify risk parameters to "force" slope 2 interest rate curve and increase slope 2 aggressiveness
+2. remove POL BUSD liquidity to increase the utilization ratio of BUSD and increase the cost of open positions.
```

Comment:
POL?

Reply:
Protocol Owned Liquidity, changing the md file to be more explicit